services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+from django.db import transaction
+from django.contrib.auth import get_user_model
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: datetime = None
+                 ) -> Order:
+    with transaction.atomic():

```

Comment:
In case all body of a method should be treated as an atomic transaction, it is better to use `transaction.atomic` as a decorator